boards/sodaq-autonomo/include/arduino_board.h

```diff
@@ -145,6 +146,26 @@ static const adc_t arduino_analog_map[] = {
     ADC_LINE(14),
 };
 
+/**
+ * @brief   RIOT GPIO mapping between Arduino pin, PWM device and channel
+ */
+typedef struct {
+    int pin;        /**< Arduino pin number */
+    int dev;        /**< PWM device index of pin */
+    int chan;       /**< PWM channel index */
+} arduino_pwm_t;
```

Comment:
I would say a bit of laziness and the fact that I was not sure where this would fit better. The mapping could depend on the underlying PWM implementation.

Reply:
The `periph_pwm` API defines PWM devices and channels of these devices. IMHO, a PWM channel is completely specified by the tupel (`dev`, `chan`) which is routed to `pin`. The `struct` should fit all. platforms. If not, the platform could extend the structure.

Reply:
exactly :-)

Reply:
I moved the struct to `periph/pwm.h`, guarded by MODULE_ARDUINO. Not sure if it's the best place. It cannot be in `arduino.hpp` since this one already includes `arduino_board.h`...